.vscode/launch.json

```diff
@@ -16,11 +16,25 @@
     {
       "type": "node",
       "request": "launch",
-      "name": "Test Core",
+      "name": "Launch HOPR Cover Traffic node",
+      "skipFiles": ["<node_internals>/**"],
+      "program": "${workspaceFolder}/packages/cover-traffic-daemon/src/index.ts",
+      "args": [
+        "--privateKey",
+        "0xcb1e5d91d46eb54a477a7eefec9c87a1575e3e5384d38f990f19c09aa8ddd332",
```

Comment:
Are there any moneys?

Reply:
No, this is a demo wallet.

Reply:
I was just kidding :-) Don't feel storing encoding private keys in the repo is a good idea but I'm good with it.